packages/router/src/index.ts

```diff
@@ -5,7 +5,6 @@
 export {
   Router,
   Route,
-  Private,
```

Comment:
Should this be removed? I'm guessing we want to let people still use `Private`, but remove it in a subsequent release?

Reply:
👍, I think we should remove the internal/ external files. It's kinda difficult to find things because of them

Reply:
Not for this PR obviously, but maybe at another stage.